http/index.php

```diff
@@ -350,7 +350,10 @@ function writeMatrix(array $glVersions, SimpleXMLElement $xml, Mesamatrix\Hints
                         </thead>
                         <tbody>
 <?php
-foreach ($xml->commits->commit as $commit) {
+$numCommits = \Mesamatrix::$config->getValue('info', 'commitlog_length', 10);
+$numCommits = min($numCommits, $xml->commits->commit->count());
+for ($i = 0; $i < $numCommits; ++$i) {
```

Comment:
I'm not sure `$xml->commits->commit` can be sliced since it's not an array. But I'll try.